src/ConnectionManager.php

```diff
@@ -139,10 +139,15 @@ protected function createConnection(string $name): MqttClientContract
         $cleanSession   = (bool) Arr::get($config, 'use_clean_session', true);
         $repository     = Arr::get($config, 'repository', Repository::class);
         $loggingEnabled = (bool) Arr::get($config, 'enable_logging', true);
+        $loggingChannel = Arr::get($config, 'log_channel', null);
 
         $settings   = $this->buildConnectionSettings(Arr::get($config, 'connection_settings', []));
         $repository = $this->application->make($repository);
         $logger     = $loggingEnabled ? $this->application->make('log') : null;
+			
+        if($logger && $loggingChannel) {
+            $logger = $logger->channel($loggingChannel);
+        }
```

Comment:
Use the same name for the variable as in config:
```suggestion
        $logChannel     = Arr::get($config, 'log_channel', null);

        $settings   = $this->buildConnectionSettings(Arr::get($config, 'connection_settings', []));
        $repository = $this->application->make($repository);
        $logger     = $loggingEnabled ? $this->application->make('log') : null;
			
        if($logger && $logChannel) {
            $logger = $logger->channel($logChannel);
        }
```